src/main/java/org/eclipse/tracecompass/traceeventlogger/LogUtils.java

```diff
@@ -296,6 +298,7 @@ public void close() {
                 return appendArgs(sb, fData).append('}').toString();
             };
             fLogger.log(new TraceEventLogRecord(fLevel, msgSupplier, time, phase, fThreadId));
+            TraceEventLoggerManager.getInstance().update(fLabel, time - fTime);
```

Comment:
In the Trace Compass implementation this is also done in the FlowScopeLog close() method. Should it be done here also?